docs/en_us/internal/testing.rst

```diff
@@ -410,12 +409,12 @@ These prerequisites are all automatically installed and available in `Devstack
 
 To run all the bok choy accessibility tests::
 
-    SELENIUM_BROWSER=phantomjs paver test_bokchoy -d accessibility
+    paver test_bokchoy --extra_args="-a 'a11y'"
 
 To run specific tests, use the ``-t`` flag to specify a nose-style test spec
-relative to the ``common/test/acceptance/accessibility`` directory::
+relative to the ``common/test/acceptance/tests`` directory::
 
-    SELENIUM_BROWSER=phantomjs paver test_bokchoy -d accessibility -t test_lms_dashboard_axs.py:LmsDashboardAxsTest.test_dashboard_course_listings_axs
+    paver test_bokchoy --extra_args="-a 'a11y'" -t test_lms_dashboard_axs.py:LmsDashboardAxsTest.test_dashboard_course_listings_axs
```

Comment:
Since that test has been refactored, the doc should be changed to give a working example.